angular-flipclock.js

```diff
@@ -58,7 +60,7 @@ angular.module('angular-flipclock', [])
         //init callbacks
         callbacks.forEach(function(callback) {
           if(attr[callback]){
-            options[callback] = function(){
+            options.callbacks[callback] = function(){
```

Comment:
it's already defined in https://github.com/stevermeister/angular-flipclock/pull/4/files#diff-02985ac8ff856f4daf9bd38da8a0ab6dR18